src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs

```diff
@@ -350,7 +347,7 @@ public async Task PayoutAsync(Balance[] balances)
                     }
 
                     logInfo = $", address={balance.Address}";
-                    var txHash = await PayoutAsync(balance);
+                    var txHash = await PayoutSingleBalanceAsync(balance);
```

Comment:
Not necessarily has to change. But I like overloaded method names as before as I used to in c# programming.

Reply:
sure, I can change it back